sdk/identity/identity/samples/typescript/src/defaultAzureCredential.ts

```diff
@@ -14,12 +14,11 @@ require("dotenv").config();
  *
  * For more information, you may go to our readme: [link](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity#defaultazurecredential)
  */
-
 export async function main(): Promise<void> {
-  const credential = new DefaultAzureCredential();
-
   const keyVaultUrl = `https://key-vault-name.vault.azure.net`;
-  const client = new KeyClient(keyVaultUrl, credential);
+
+  let credential = new DefaultAzureCredential();
+  let client = new KeyClient(keyVaultUrl, credential);
```

Comment:
Interesting. I'm ok with this, but I wanted to make sure this was intentional.

Reply:
Not intentional, I was using this file as a playground as I writing the samples and forgot to reset things. Good catch, thanks!